client/components/editor.vue

```diff
@@ -185,6 +185,14 @@
               // TODO: Error handling.
             });
           }
+          // update user current position
```

Comment:
Is this really the best place to do so? Is not there any event ProseMirror sends when user moves around? Because this is triggered only when content changes. Not when user moves around?

Reply:
I think that's the best place because according to ProseMirror reference they treat change of selection (and cursor position) as a state change, and all state changes are handled by that callback.

Reply:
Hm, do they communicate those state changes? Are any new steps generated on those state changes?

Reply:
So, probably this is what `sendable` is then. It removes all those non-step state changes?

Reply:
Yeah, `sendable` tells us if current state changes are steps that should be sent over. But I think selections do not make `sendable` be true.